docs/development/Synchronizing-GitLab-Branch.adoc

```diff
@@ -0,0 +1,46 @@
+= Synchronizing GitLab Branch =
+
+== Overview ==
+
+This page describes the procedure to synchronize a branch from an upstream repository
+to a GitLab repository.
+
+== Creating Access Token ==
+
+In the GitLab repository create a project access token with a **write_repository** permission.
+
+See link:https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#creating-a-project-access-token[Creating a project access token].
+
+== Configuring Synchronization ==
+
+In the GitLab repository create the following variables:
+
+* `UPSTREAM_URL`: The URL of the upstream repository.
+* `ACCESS_TOKEN`: The value of the access token.
+** Select **Mask variable** to protect the access token.
```

Comment:
It is worth noting here to use this on *unprotected* branches you need to uncheck "Protect variable"

Reply:
Yes, I've updated the doc.